src/cli.js

```diff
@@ -10,6 +10,9 @@ const packageJson = require('./../package.json');
 const files = require('./utils/files');
 const inquirer = require('./inquirer');
 const utils = require('./utils/utils');
+const secure = require('./secure/secure');
+const keytar = require('keytar');
```

Comment:
add the module to package.json, to prevent this always use npm install --save package_name